kornia/geometry/transform/flips.py

```diff
@@ -139,4 +139,5 @@ def vflip(input: torch.Tensor) -> torch.Tensor:
 
     """
 
-    return torch.flip(input, [-2])
+    h = input.shape[-2]
+    return input[..., torch.arange(h - 1, -1, -1), :]
```

Comment:
Will this still work if input is on GPU and torch.arange output is CPU?

Reply:
this was fixed https://github.com/kornia/kornia/blob/master/kornia/geometry/transform/flips.py#L149

Did you encounter any issues around this ?

Reply:
Ah, okay. Then it should be fine!